paris/src/testDebug/java/com/airbnb/paris/proxies/TextViewStyleApplierTest.kt

```diff
@@ -54,6 +56,19 @@ class TextViewStyleApplierTest {
         assertFalse(view.compoundDrawables[0].bounds.isEmpty)
     }
 
+    @Test
+    fun fontFamily_sansSerif() {
+        applier.apply(R.style.Test_TextViewStyleApplier_FontFamily_String)
+        assertEquals(Typeface.create("sans-serif", Typeface.NORMAL), view.typeface)
+    }
+
+    @Test
+    @Ignore("Robolectric can't handle font resources - see https://github.com/robolectric/robolectric/issues/3590")
+    fun fontFamily_fontReference() {
+        applier.apply(R.style.Test_TextViewStyleApplier_FontFamily_Resource)
+        assertEquals(ResourcesCompat.getFont(context, R.font.roboto_regular), view.typeface)
```

Comment:
I think shadowing `ResourcesCompat` might be worth it here. `fontFamily` appears to be a particularly tricky attribute so I'd feel more confident if it was fully tested.

Alternatively, although I've been trying to move away from them, we could make it an instrumented tests (that runs on an actual device or emulator) so that we don't need to rely on Robolectric.